Sources/ShapedArray/Operations/pack.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  File.swift
+//  
+//
+//  Created by Jaap Wijnen on 10/08/2022.
+//
+
+extension ShapedArray {
+    static func pack(_ values: [ShapedArray], axis: Int = 0) -> ShapedArray {
```

Comment:
We don't need to extract primitive operations into a dedicated internal function. S4TF uses this convention with `_Raw`, but the Swift Stdlib just uses module-internal initializers. This could become `ShapedArray.init(stacking:alongAxis:)` like so:

https://github.com/s4tf/s4tf/blob/a31f79a7a8e92ff708b1034286a23510b23af5f3/Sources/TensorFlow/Initializers.swift#L152-L156

The array literal initializer calls into the more generic `init(stacking:alongAxis:)`, which behaves just like `_Raw.pack(_:axis:)` from S4TF.

Reply:
We could use `arrays` for the interior argument label of `init(stacking:alongAxis)`, which matches S4TF's `stacking tensors`. It seems kind of restrictive, but using S4TF as precedent gives solid ground for making decisions that could go either way.

Reply:
Also the method should be `@inlinable` just like S4TF.

Reply:
Isn't pack a super generic operation used all the time though?